src/pycram/world_concepts/world_object.py

```diff
@@ -487,7 +501,7 @@ def get_link_axis_aligned_bounding_box(self, link_name: str) -> AxisAlignedBound
         :param link_name: The name of the link.
         :return: The axis aligned bounding box of the link.
         """
-        return self.links[link_name].get_axis_aligned_bounding_box()
+        return self.links[link_name].get_bounding_box()
```

Comment:
This could Return a bounding box that is Not Anis aligned which is confusing given the Method Name

Reply:
fixed it